docs/docs/communicate/js-from-fable.md

```diff
@@ -188,8 +188,8 @@ we could use the same method we used with `alert.js`:
   open Fable.Core.JsInterop // needed to call interop tools
 
   type ICanvas =
-    abstract drawSmiley: (id:string) -> unit
-    abstract drawBubble: (id:string) -> unit
+    abstract drawSmiley: string -> unit
+    abstract drawBubble: string -> unit
```

Comment:
Ah, you're right. Parentheses are not allowed here (which is not intuitive, but anyways). I think we could keep the param name, but for the purpose of this sample it may be better to remove it. Thanks!